drivers/oled/oled_driver.h

```diff
@@ -206,6 +206,10 @@ void oled_pan(bool left);
 void oled_write_raw(const char *data, uint16_t size);
 void oled_write_raw_byte(const char data, uint16_t index);
 
+// Sets a specific pixel on or off
+// Coordinates start at top-left and go right and down for positive x and y
+void oled_write_pixel(int16_t x, int16_t y, bool on);
```

Comment:
Why `int16`? I don't know of any OLED displays that go > 256px.

Reply:
The idea was to allow pixels to be set off screen, but I suppose that check is better done in userland. Will update shortly.

On another note, what do you think about including [these 2 functions](https://github.com/GauthamYerroju/qmk_firmware/blob/oled_starfield/keyboards/kyria/keymaps/gotham/oled_animations/common.c) in the driver?

Reply:
Drawing offscreen is not very practical, at least on AVR; a 128x64 display takes up 1024 bytes in RAM, which very nearly fills up the 1.25kB available on the 32U4.

Reply:
I also note that negative x/y probably would not work with the calculations here anyway.

Reply:
Good point about the calculations!